config/environments/development.rb

```diff
@@ -30,13 +30,6 @@
   # Expands the lines which load the assets
   config.assets.debug = true
 
-  # Raise exception on mass assignment protection for Active Record models
-  config.active_record.mass_assignment_sanitizer = :strict
```

Comment:
If it's enabled in test, should it also be enabled here in dev and also in prod?